e2e-tests/functions

```diff
@@ -36,11 +36,7 @@ if [ -f "$conf_dir/cloud-secret.yml" ]; then
 fi
 
 if oc get projects; then
-	# Guessing Openshift version from master node API version
-	case "$(oc get nodes --no-headers=true | grep master | head -n1 | grep -Eo 'v[0-9]+\.[0-9]+')" in
-		"v1.11") OPENSHIFT=3.11 ;;
-		*) OPENSHIFT=4 ;;
-	esac
+	OPENSHIFT=4
```

Comment:
Why this change? I mean why stop "# Guessing Openshift version...".

Reply:
We do not support Openshift 3.11 https://github.com/percona/percona-server-mongodb-operator/pull/973#discussion_r925445669